WordPress/Classes/ViewRelated/NUX/LoginEpilogueUserInfo.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+/// A simple container for the user info shown on the login epilogue screen.
+///
+struct LoginEpilogueUserInfo {
+    var username = ""
+    var fullname = ""
```

Comment:
I'm ok with `username` casing but `fullName` should have that capital N, just like `fullNameLabel` elsewhere. Either that, or I'm super wrong and you should lowercase both Ns.

Big decision.